src/Workspaces/Core/Portable/Workspace/Solution/DocumentInfo.cs

```diff
@@ -9,8 +9,15 @@ namespace Microsoft.CodeAnalysis
     /// <summary>
     /// A class that represents all the arguments necessary to create a new document instance.
     /// </summary>
+    [System.Diagnostics.DebuggerDisplay("{GetDebuggerDisplay()}")]
     public sealed class DocumentInfo
     {
+
+        private string GetDebuggerDisplay()
+        {
+            return nameof(FilePath) + " = " + FilePath;
```

Comment:
@AdamSpeight2008 Why don't you make this return `Name` when `FilePath` is null.

Also - style nits:

1. Don't start the class with a blank line.
2. Put that method at the end of the type, instead of the beginning (we don't tend to have methods above fields).